internal/db/project.go

```diff
@@ -13,6 +14,13 @@ func CreateProject(ctx context.Context, tx pgx.Tx, project int64, packPatterns [
 		INSERT INTO dl.projects (id, latest_version, pack_patterns)
 		VALUES ($1, 0, $2)
 	`, project, packPatterns)
+
+	var projectExistsError = errors.New("ERROR: duplicate key value violates unique constraint \"projects_pkey\" (SQLSTATE 23505)")
+
+	if err != nil && err.Error() == projectExistsError.Error() {
+		return fmt.Errorf("project id already exists")
+	}
```

Comment:
Should we just return `nil` and noop instead?

Reply:
Well that can be dangerous id the `packPatterns` is different, in that case you could think you are making a project with one set of values and instead another was already created.

Reply:
right, makes sense 👍